src/routes/+layout.svelte

```diff
@@ -1,21 +1,3 @@
-<script context="module" lang="ts">
-  import configure from '$lib/config/config';
-
-  configure({
-    clientId: 'WebOAuthClient',
-    // redirectUri: 'https://crbrl.ngrok.io/callback',
-    redirectUri: 'https://localhost:3000/callback',
-    scope: 'openid profile me.read',
-    serverConfig: {
-      baseUrl: 'https://openam-crbrl-01.forgeblocks.com/am/',
-      // baseUrl: 'https://crbrl.ngrok.io/proxy/',
-      timeout: 5000,
-    },
-    realmPath: 'alpha',
-    tree: 'Login',
-  });
-</script>
-
 <svelte:head>
```

Comment:
There's a lot of these modifications to all the route files. A lot of this was driven by SvelteKit changes and a provided migration script. Then, I had to just double check everything after the migration script ran.